ansible/roles/docker-registry/templates/config.yml.j2

```diff
@@ -0,0 +1,8 @@
+version: 0.1
+storage:
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:5000
+  secret: thisismysecrettherearemanylikeitbutthisoneismine
```

Comment:
Can't store this.. will need to use an ansible var and make sure it's available.